include/net/nrf_cloud_location.h

```diff
@@ -92,8 +105,8 @@ typedef void (*nrf_cloud_location_response_t)(const struct nrf_cloud_location_re
  *                 To omit a request parameter, use the appropriate
  *                 `NRF_CLOUD_LOCATION_WIFI_OMIT_` define.
  *                 Can be NULL if cell info is provided.
- * @param request_loc If true, cloud will send location to the device.
- *                    If false, cloud will not send location to the device.
+ * @param config   Optional configuration of request. If NULL, fall back to default
+ *                 which is do_reply = true, hi_conf = false, and fallback = true.
  * @param cb Callback function to receive parsed location result. Only used when
  *           request_loc is true. If NULL, JSON result will be sent to the cloud event
```

Comment:
`request_loc` is not there anymore so need to refer to the `config->do_reply` etc. and the comment should take care of config=NULL case too which means this callback would be valid.

Reply:
The comment does take care of config=NULL.
```
* @param config   Optional configuration of request. If NULL, fall back to default
 *                 which is do_reply = true, hi_conf = false, and fallback = true.

Reply:
OK, missed that.